src/main/php/scriptlet/xml/workflow/Wrapper.class.php

```diff
@@ -309,4 +312,20 @@ protected static function isEmptyFileUpload($value) {
       isset($value['size'])     && 0 === $value['size']
     );
   }
+
+  /**
+   * Check if the provided value is file upload field
+   *
+   * @param   var value
+   * @return  bool
+   */
+  protected static function isFileUpload($value) {
+    return (
+      array_key_exists('name', $value) &&
```

Comment:
There are optional keys in file that can be null as value. The logic here is to check that all keys exists to identify a file.

```php
$a = array('key1'=> 'test', key2' => null);
isset($a['key2']);             // false
array_key_exists('key2', $a);  // true
```